book/src/rpc/mod.rs

```diff
@@ -1,14 +1,15 @@
-use futures::{future, prelude::*};
+pub mod models;
+pub mod server;
+pub mod service;
+
 use std::io;
 use std::net::SocketAddr;
```

Comment:
```suggestion
use std::{
    io,
    net::SocketAddr
};
```

Reply:
Dislike it, since it takes up more lines.